### PR TITLE
2850: POI filter button shines through language selection overlay

### PR DESCRIPTION
**Pull request by bahaaTuffaha**
### Short description

POI filter button above the  language selection overlay.

### Proposed changes

- Changed the z-index for OverlayContainer from 1 to 0 Located at MapView component.

### Side effects

None

### Resolved issues

Fixes: #2850 

---

### CODE REVIEW

File: release-notes/unreleased/2850-POI-filter-button-shines-through.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: 2850
+show_in_stores: true
```
Comment (LeandraH) [inline release-notes/unreleased/2850-POI-filter-button-shines-through.yml:2]: Since this is only a web issue, it doesn't need to be shown in the stores :)

**Review by LeandraH — APPROVED** [review]
Very nice, thanks! Tested in Firefox

File: release-notes/unreleased/2850-POI-filter-button-shines-through.yml (hunk 1 above)
Comment (steffenkleinle) [reply release-notes/unreleased/2850-POI-filter-button-shines-through.yml]: Accordingly, no German translation is necessary :) We only need German translations for ios/android issues that are shown in the stores.

File: web/src/components/MapView.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const OverlayContainer = styled.div`
   display: flex;
   padding: 12px 8px;
   flex: 1;
-  z-index: 1;
+  z-index: 0;
```
Comment (steffenkleinle) [inline web/src/components/MapView.tsx:46]: As the default is zero, this could be omitted, correct?
```suggestion
```

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox

File: web/src/components/MapView.tsx (hunk 2 above)
Comment (LeandraH) [reply web/src/components/MapView.tsx:46]: Good point!

**Comment by LeandraH** [comment]
@bahaaTuffaha Though in this case, please still remove the `z-index` and test it once more before merging 🙂

**Pull request merged**